src/deepforest/visualize.py

```diff
@@ -517,11 +517,11 @@ def plot_results(results,
         image_path = os.path.join(root_dir, results.image_path.unique()[0])
         image = np.array(Image.open(image_path))
 
-        # Drop alpha channel if present and warn 
+        # Drop alpha channel if present and warn
         if image.shape[2] == 4:
```

Comment:
We should be able to use the utility we created to verify that the image exists, is not empty, and has at least 3 channels. If it has more than 3 (e.g., an alpha channel), we can use drop the extra channel.